client/js/shout.js

```diff
@@ -382,6 +382,28 @@ $(function() {
 		}
 	}
 
+	if ($.cookie("timestamp")) {
+		$("#timestamp").val($.cookie("timestamp")).change();
+	} else {
+		$("#timestamp").val("HH:mm").change();
+	}
+
+	$("#timestamp").on("change", function() {
+		var timestamp = $(this).val();
+		Handlebars.unregisterHelper("tz");
```

Comment:
Ah wait, yes, of course. I can do that, but I would still need to have the helper register be out of it's existing file, which isn't ideal, as that's less modularising rather than more :-P